pkgs/development/python-modules/pyzufall/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, python, buildPythonPackage, nose, future, coverage }:
+
+buildPythonPackage rec {
+  pname = "PyZufall";
+  version = "0.13.2";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "mirror://pypi/${builtins.substring 0 1 pname}/${pname}/${name}.tar.gz";
+    sha256 = "1jffhi20m82fdf78bjhncbdxkfzcskrlipxlrqq9741xdvrn14b5";
+  };
+
+  # Cannot build wheel otherwise (zip 1980 issue)
+  SOURCE_DATE_EPOCH=315532800;
+
+  # disable tests due to problem with nose
+  # https://github.com/nose-devs/nose/issues/1037
+  doCheck = false;
```

Comment:
it had doctests in the code and test.py, which is not included in the archive in the current release, but everytime the tests fail even before the first test is executed (see linked issue)